bootstrap.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# make bash behave
+set -euo pipefail
+IFS=$'\n\t'
+
+# call PostgreSQL's ENTRYPOINT script
+./docker-entrypoint.sh postgres
```

Comment:
This isn't going to work, since in Docker, the `ENTRYPOINT` basically defines the "main process" of the image. It's never expected to exit.

I think we should do our `root` stuff first, then continue relying on the PostgreSQL's image's script functionality for the rest. The final line of the shell script should be an `exec` call to `docker-entrypoint.sh`, passing on any arguments. That way we naturally segue into the `ENTRYPOINT` logic in this process rather than forking or shelling out, etc.